cmd/utils.go

```diff
@@ -58,4 +59,35 @@ func preRunConfig(cmd *cobra.Command, args []string) {
 	if !configFileUsed {
 		log.Info("config file not found, proceeding without it")
 	}
+
+	ArtifactPath()
```

Comment:
Instead of this, just use ArtifactPath() in check_container.go and check_operator.go where resultsFilenameWithExtension is being called.